playwright.config.ts

```diff
@@ -0,0 +1,110 @@
+import type { PlaywrightTestConfig, expect } from '@playwright/test';
+import { devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
```

Comment:
We should also provide a [`webServer` config][1] for the e2e tests to work on their own (without having to manually start a webserver). Something like:

```suggestion
const config: PlaywrightTestConfig = {
  webServer: {
    command: 'npm start',
    port: 8080,
  },
```

[1]: https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests

Reply:
Added